bitnami/kafka/Chart.yaml

```
@@ -29,4 +29,4 @@ name: kafka
 sources:
   - https://github.com/bitnami/bitnami-docker-kafka
   - https://kafka.apache.org/
-version: 15.1.0
+version: 15.1.1
```

You're adding a new parameter, therefore you should bump the minor version:
```suggestion
version: 15.2.0
```